composer.json

```
@@ -24,15 +24,15 @@
 		"source": "https://github.com/infinum/eightshift-frontend-libs"
 	},
 	"require": {
-		"php": "^7.4 || <8.0",
+		"php": "^7.4 || >=8.0",
```

```suggestion
		"php": ">=7.4",
```
we can always change it to `>=7.4 <= 8.2` for example if 8.3 adds breaking changes

---

@dingo-d what do you think?

---

Yeah, that should work 👍🏼